oak_functions/README.md

```diff
@@ -8,18 +8,23 @@ sandboxing, and allows exposing metrics in a Differentially Private way.
 At its core, Oak Functions consists of a **trusted runtime** compiled into a
 server binary
 ([`oak_functions_loader`](https://github.com/project-oak/oak/tree/main/oak_functions/loader))
-that, for each incoming client gRPC request, executes an untrusted workload that
-operates on the request payload, and produces a response which is sent back to
-the same client.
+that, for each incoming client gRPC request, executes a workload that operates
+on the request payload, and produces a response which is sent back to the same
+client.
 
-The Oak Functions trusted runtime ensures that the untrusted workload may not
-violate the confidentiality of the client request data, preventing observers
-from learning anything about the request.
+The Oak Functions trusted runtime ensures that the workload may not violate the
+confidentiality of the client request data, preventing observers from learning
+anything about the request. Note that since the Oak Functions trusted runtime
+guarantees the confidentiality and integrity of the data, the workload can
+potentially be untrusted.
```

Comment:
I think there is some repetition in this paragraph. Could we shorten to:

"The Oak Functions trusted runtime ensures that the workload does not violate the confidentiality of the client request data---even for workloads, which are not trusted by the Oak Functions trusted runtime."

Also, the user will need to trust the workload at the end of the day?

Reply:
Yes the intention was that it does not need to be trusted by the client (nor by the host).

Reply:
Thanks :) Changed to sentence to the following to better clarify it. 

```
As a result, the workload can potentially be
untrusted, meaning that neither the client nor the host needs to know the
identity of the workload or trust it.
```